apps/opencs/view/render/object.cpp

```diff
@@ -140,7 +140,7 @@ void CSVRender::Object::update()
     if (light)
     {
         bool isExterior = false; // FIXME
-        SceneUtil::addLight(mBaseNode, light, Mask_ParticleSystem, Mask_Lighting, isExterior);
+        SceneUtil::addLight(mBaseNode, light, isExterior);
```

Comment:
Here you have also removed the Mask_ParticleSystem and Mask_Lighting. What's the purpose of that?

Reply:
Read the PR description. There are answers for all your questions.